imagemeta/bmp.go

```diff
@@ -29,7 +29,13 @@ func DecodeBmpMeta(r io.Reader) (Meta, error) {
 
 	if infoSize >= 40 {
 		width = int(binary.LittleEndian.Uint32(tmp[18:22]))
-		height = int(binary.LittleEndian.Uint32(tmp[22:26]))
+		signedHeight := int32(binary.LittleEndian.Uint32(tmp[22:26]))
+		// height can be negative in Windows bitmaps
+		if signedHeight < 0 {
+			height = int(-signedHeight)
+		} else {
+			height = int(signedHeight)
+		}
```

Comment:
Here's a bit shorter version:

```suggestion
		height := int(int32(binary.LittleEndian.Uint32(tmp[22:26])))
		// height can be negative in Windows bitmaps
		if height < 0 {
			height = int(-height)
		}
```

Also, it seems like the same trick should be applied to 16-bit height. And we can move the negative height check out of this `if`.